doc/api/toolkits/mplot3d/view_angles.rst

```diff
@@ -0,0 +1,40 @@
+.. _toolkit_mplot3d-view-angles:
+
+*******************
+mplot3d View Angles
+*******************
+
+How to define the view angle
+============================
+
+The position of the viewport "camera" in a 3D plot is defined by three angles:
+elevation, azimuth, and roll. From the resulting position, it always points
+towards the center of the plot box volume. The angle direction convention is
+shared with
```

Comment:
Minor, but i think emphasizing the angle names makes sense.
```suggestion
*elevation*, *azimuth*, and *roll*. From the resulting position, it always
points towards the center of the plot box volume. The angle direction
convention is shared with
```

Also, could we say

"The angle direction is a common convention and e.g. shard with ..."